test/sync-profile.test.js

```diff
@@ -0,0 +1,182 @@
+process.env.REACT_APP_SIGNALHUB_URLS = 'localhost:8080'
+const { REACT_APP_SIGNALHUB_URLS } = process.env
+
+import * as common from 'masq-common'
+import signalhub from 'signalhubws'
+import swarm from 'webrtc-swarm'
+
+const { expect } = require('chai')
+const Masq = require('../src/lib/masq').default
+const SyncProfile = require('../src/lib/sync-profile').default
+
+const { dbExists } = common.utils
+const { genAESKey, exportKey } = common.crypto
+
+const waitForSync = async (db1, db2) => {
```

Comment:
A function similar to this one could help us detect if a DB is synced with another one. We will move this to another file